### ISSUE TITLE
syntax error: invalid regular expression `/^i-???????$/:` nothing to repeat

### ISSUE THREAD

**Issue by ericpaulsen** [coder/vscode-coder#108]
a customer reported seeing this error when opening a connection and reconnecting.

```
Failed to open workspace
SyntaxError: Invalid regular expression: /^i-???????? i-?????????????????$/: Nothing to repeat
```

![image](https://github.com/coder/vscode-coder/assets/9683576/a378d30a-0e13-4a5d-bd9c-6494d0033591)

here's their VS Code, Coder Remote, and client OS configs:

```
# VS Code
Version: 1.79.0
Commit: b380da4ef1ee00e224a15c1d4d9793e27c2b6302
Date: 2023-06-07T14:29:00.206Z
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin x64 22.2.0

# macOS
ventura 13.1

# Coder Remote
v0.1.19
```

**Comment by ohimors** [comment]
What's the status of the fix for this issue?

![Screenshot 2023-09-27 at 3 02 41 PM](https://github.com/coder/vscode-coder/assets/452292/be6676e9-c690-41ab-9b9c-6245bf4d5776)

Version: 1.82.2
Commit: abd2f3db4bdb28f9e95536dfa84d8479f1eb312d
Date: 2023-09-14T05:59:47.790Z (2 wks ago)
Electron: 25.8.1
ElectronBuildId: 23779380
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin x64 22.6.0

**Comment by ericpaulsen** [comment]
@ohimors thank you for following up. it appears this bug was fixed and then reintroduced in the latest release of the extension. can you confirm what version of the extension you experienced this on?

cc: @code-asher

**Comment by ohimors** [comment]
Coder Remote v0.1.23 is installed.

**Comment by code-asher** [comment]
I am aiming to take a look at this next week.

**Comment by code-asher** [comment]
Fix released in 0.1.25.  Let me know if there are still issues.
